sshd-sftp/src/main/java/org/apache/sshd/client/subsystem/sftp/impl/SftpRemotePathChannel.java

```diff
@@ -524,13 +527,13 @@ protected void endBlocking(Object actionHint, boolean completed)
      * @param  reqModes    The required modes - ignored if {@code null}/empty
      * @throws IOException If channel not open or the required modes are not satisfied
      */
-    private void ensureOpen(Collection<SftpClient.OpenMode> reqModes) throws IOException {
+    private void ensureOpen(Collection<OpenMode> reqModes) throws IOException {
```

Comment:
Let's make it protected in accordance with our policy to allow maximum flexibility to users.

Reply:
Sure, but that's not part of my changes, so I'd rather use a different commit to fix issues in the existing code rather than mixing both.

Reply:
+1